packages/aws-cdk-lib/aws-cloudwatch/lib/metric.ts

```diff
@@ -485,7 +485,7 @@ export class Metric implements IMetric {
   public toAlarmConfig(): MetricAlarmConfig {
     const metricConfig = this.toMetricConfig();
     if (metricConfig.metricStat === undefined) {
-      throw new Error('Using a math expression is not supported here. Pass a \'Metric\' object instead');
+      throw new cdk.UnscopedValidationError('Using a math expression is not supported here. Pass a \'Metric\' object instead');
```

Comment:
these are all unscoped because `Metric` isn't a Construct. We'd have to send in `scope: Construct` as a property to all metrics everywhere to get this to be scoped